libs/explorers/styles/src/lib/_variables.scss

```diff
@@ -47,6 +92,90 @@ $extra-colors: (
   }
 }
 
+// -------------------------------------------------------------------------- //
+// Fonts
+// -------------------------------------------------------------------------- //
+
+$font-family-primary: 'DM Sans', sans-serif;
+$headings: (
+  h1: (
+    font-size: 1.75rem /* 28px */,
+    font-weight: 700,
+    line-height: 2rem /* 32px */,
+  ),
+  h2: (
+    font-size: 1.5rem /* 24px */,
+    font-weight: 700,
+    line-height: 1.75rem /* 28px */,
+  ),
+  h3: (
+    font-size: 1.25rem /* 20px */,
+    font-weight: 500,
+    line-height: 1.5rem /* 24px */,
+  ),
+  h4: (
+    font-size: 1rem /* 16px */,
+    font-weight: 500,
+    line-height: 1.25rem /* 20px */,
+  ),
+  h5: (
+    font-size: 0.875rem /* 14px */,
+    font-weight: 500,
+    line-height: 1.25rem /* 0px */,
+  ),
+  h6: (
+    font-size: 0.75rem /* 12px */,
+    font-weight: 500,
+    line-height: 1.125rem /* 18px */,
+  ),
+);
+$font-sizes: (
+  xs: (
+    font-size: 0.75rem /* 12px */,
+    line-height: 1.125rem /* 18px */,
+  ),
+  sm: (
+    font-size: 0.875rem /* 14px */,
+    line-height: 1.25rem /* 20px */,
+  ),
+  md: (
+    font-size: 1rem /* 16px */,
+    line-height: 1.5rem /* 24px */,
+  ),
+  lg: (
+    font-size: 1.125rem /* 18px */,
+    line-height: 1.5rem /* 24px */,
+  ),
+  xl: (
+    font-size: 1.25rem /* 20px */,
+    line-height: 1.625rem /* 26px */,
+  ),
+  xxl: (
+    font-size: 1.375rem /* 22px */,
+    line-height: 1.875rem /* 30px */,
+  ),
+);
```

Comment:
Although I see the benefit of potentially creating these, I feel like we should consider just standardizing on px (although I love rem).  Also, I feel like font sizing might be best colocated in each component and if needed, passed in as an Input parameter unless there is a strong connection between components requiring a particular size.  We can take this discussion offline if necessary.